component.go

```diff
@@ -14,6 +16,40 @@ type ComponentTypeConnection struct {
 	TotalCount int             `json:"totalCount" graphql:"-"`
 }
 
+func (s *ComponentType) GetProperties(client *Client, v *PayloadVariables) (*PropertyDefinitionConnection, error) {
```

Comment:
This is the pattern we use for connection types because it limits how much data we pull back when doing "list" api calls.